tests/workflows/snvphylnfc.nf.test

```diff
@@ -145,4 +145,28 @@ nextflow_workflow {
 
     }
 
+    test("disabling snv density filtering") {
+
+        when {
+            params {
+                input = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv"
+                refgenome = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/reference.fasta"
+                outdir = "results"
+                skip_density_filter = true
+            }
+            workflow {}
+        }
+
+        then {
+            assert workflow.success
+            assert path("$launchDir/results").exists()
+
+            // check that no "_filtered_density.txt" files exist:
+            assert path("$launchDir/results/consolidate/SAMPLE1_filtered_density.txt").exists() == false
```

Comment:
Can you add a check in the existing end-to-end tests to make sure these files do exist in the case where density filtering is not skipped?

Reply:
Added in 3891cce31a1881bf07c4f52b68cd35eb9b96489b